stack/defaultstackpolicy/blocked.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I have one question where those files will land when we will distribute `perun` as a binary?